src/reducers/entities/posts.js

```diff
@@ -31,6 +31,35 @@ function handleReceivedPost(posts = {}, postsInChannel = {}, action) {
     return {posts: nextPosts, postsInChannel: nextPostsForChannel};
 }
 
+function handleRemovePendingPost(posts = {}, postsInChannel = {}, action) {
+    const postId = action.data.id;
+    const channelId = action.data.channel_id;
+
+    const nextPosts = {
+        ...posts,
+    };
+
+    Reflect.deleteProperty(nextPosts, postId);
+
+    let nextPostsForChannel = postsInChannel;
+
+    // Only change postsInChannel if the order of the posts needs to change
+    if (!postsInChannel[channelId] || postsInChannel[channelId].indexOf(postId) !== -1) {
```

Comment:
Can you change indexOf for includes?

Reply:
I have changed it here, and allover the file.